### DISCUSSION TITLE
Usability improvements - Text scaling, pin/bookmarks on page index

### DISCUSSION THREAD

**Discussion by msinfo32github** [redimp/otterwiki#109] [Ideas]
This project is exactly what I've been looking to find for ages, it's amazing!

I really think that these two features would make it perfect.

## Scaling text size

- Currently, Otter wiki does not support scaling text size. This means that if you choose to have a smaller browser zoom scale (e.g. scaling to 70%) it does not look as 'crisp'. Mainly though, when moving across different pages this **page zoom is not kept**.

## Pin/bookmarking pages

- Although I haven't used Otter wiki for very long, I think that adding bookmark pages/pinned pages for easy access, e.g. on the page index at the side of the page would make navigating often accessed pages easier. Alternatively, this could maybe be a 'Recent' list of recently accessed wiki pages.

Aside from those two things, Otter Wiki is perfect for myself personally and I hope to never need to replace it, thank you for your work on the project!

**Comment by msinfo32github** [comment]
One little improvement/option (however less of an importance) is the page index naming with capitalisation.

See the page title here:

![Screenshot_20240506_000351](https://github.com/redimp/otterwiki/assets/37910790/d12752b4-fbac-461f-af97-6343504bc250)


See the page index title here:

![image](https://github.com/redimp/otterwiki/assets/37910790/9fcabbe2-6d94-4700-bc43-465040879f89)


(VM vs Vm)

**Comment by redimp** [comment]
Hey @msinfo32github, thanks for your feedback.

1.) Text scaling

I've tested the text scaling with Firefox, Chrome and Brave: The scaling is preserved when browsing throught the wiki, see the recording: 

https://github.com/redimp/otterwiki/assets/112056/ed6ff0f7-a0d9-453a-afb9-2a34fea9f319

With which browser (and OS) does it break for you?

2.) Pin/Bookmark pages

I think this would be a very useful feature. Where would you expect to find your pinned pages? In the sidebar? In an extra Menu?

3.) Case in the sidebar not matching header

Confirmed, will check into it.

**Comment by msinfo32github** [comment]
Thanks for the reply, @redimp.

1 - I'm using LibreWolf and I believe this is actually caused by the resistFingerprinting it uses. However, I still think it may be a nice feature, e.g. for using across multiple devices?
2 - My thinking behind this feature is that it would be easily accessible on the sidebar is this can be easily shown/hidden. I've created a basic mock-up of the idea here:

![Screenshot_20240514_171401](https://github.com/redimp/otterwiki/assets/37910790/f73684ec-728d-44d2-a340-84807e5bcc98)
